docs/builds/guides/integration/configuration.md

```diff
@@ -64,8 +64,59 @@ ClassicEditor.builtinPlugins.map( plugin => plugin.pluginName );
 
 ## Adding features
 
+### Adding complex features
+
 As CKEditor builds do not include all possible features, the only way to add more features to them is to {@link builds/guides/development/custom-builds create a custom build}.
 
+### Adding simple (standalone) features
+
+There is an exception to every rule. Although it is impossible to add plugins that have dependencies to {@link api/core `@ckeditor/ckeditor5-core`} or {@link api/engine `@ckeditor/ckeditor5-engine`} (that includes nearly all existing official plugins) without rebuilding the build, it is still possible to add simple, **dependency-free** plugins.
+
+You can do that using the {@link module:core/editor/editorconfig~EditorConfig#extraPlugins `config.extraPlugins`} configuration. The {@link module:core/plugin~PluginInterface plugin interface} allows plugins to be simple functions and you can define them in just a few lines, for instance:
+
+```js
```

Comment:
Do we post examples of "ES5" classes in other parts of docs?

Reply:
That's not an ES5 class, it's a mention that you can write a simple function plugin. Perhaps if it was named `myPlugin()` it would be clearer, but I'd keep it like this.